databox/api/config/validator/validation.yaml

```diff
@@ -127,6 +127,7 @@ App\Entity\Core\RenditionDefinition:
           - workspace
           - class.workspace
           - parent.workspace
+    - App\Validator\ValidRenditionDefinitionConstraint: ~
```

Comment:
You validate a property only, not multiple values of the entity.
It should be placed on the property directly.